torch/_tensor_docs.py

```diff
@@ -3428,7 +3428,16 @@ def callable(a, b) -> number
 
 add_docstr_all('stft',
                r"""
-stft(frame_length, hop, fft_size=None, return_onesided=True, window=None, pad_end=0) -> Tensor
+stft(n_fft, hop_length=None, win_length=None, window=None,
+ center=True, pad_mode='reflect', normalized=False, onesided=True) -> Tensor
+
+See :func:`torch.stft`
+""")
```

Comment:
nit: What is the update here for `stft`? I would not expect this to change.

Reply:
removed this change.